### DISCUSSION TITLE
Share the latest version of Spotify Windows

### DISCUSSION THREAD

**Discussion by amd64fox** [SpotX-Official/SpotX#7] [General]
I have created an [auto-updatable document.](https://cutt.ly/8EH6NuH)
The script itself collects new versions from different sources, and to the right of the list there is a link to a [form](https://docs.google.com/forms/d/e/1FAIpQLSdqIxSjqt2PcjBlQzhvwqc4QckfWuq5qqWsrdpoTidQHsPGpw/viewform) that gives anyone who wants to add the new version himself, if he has one.
All links to the installation clients lead to the original Spotify server, which is absolutely safe.

**Comment by 54m4d** [comment]


**Reply by amd64fox** [reply]
@fosam 
Hi, I agree, my region (Russia) also receives updates very late. Therefore, I created a theme, perhaps there will be people with a different region and will throw off the latest version.

By the way, what region do you have if it's not a secret?

**Reply by amd64fox** [reply]
Thank you very much for the information.

There is a line on the main page of this repository `Last tested version`, If your version is higher than the one indicated there, then throw it here in the chat, I will be grateful to you for that.

This way I will be able to fix possible errors almost imperceptibly, if there are any.

**Reply by amd64fox** [reply]
No, this is not beta, in some regions a new, stable version has already been released.

The main thing is to find out the full name of the version, and then you can download it through [this tool](https://github.com/nick-botticelli/SpotifyUpgradeFinder).

For example, here is a link to the new version [1.1.67.584](https://upgrade.scdn.co/upgrade/client/win32-x86/spotify_installer-1.1.67.584.gc0d1ad42-14.exe) which i got.

**Reply by amd64fox** [reply]
I'm currently testing link hijacking, but so far I can't get it out, there is not enough knowledge in the `JavaScript`.

If I succeed I will unsubscribe here

**Reply by amd64fox** [reply]
> The thing is if my region is limited to `1.1.66.580.gbd43cbc9-a` how would one intercept the URL for `1.1.67.584.gc0d1ad42`

As an option, use a VPN to disguise yourself as a different region.

**Reply by amd64fox** [reply]
I know India is one of that number

**Reply by amd64fox** [reply]
Try to check with this person through which update channel he received this version 1.67. It is not enough to change the region through a Vpn.
Perhaps this should also be taken into account:
- the country of your account
- client's language

**Reply by amd64fox** [reply]
Yes, I checked it 2 days ago, it works

**Reply by amd64fox** [reply]
It is impossible to get the update at this link the earliest, here the channel is completely different, I think those who get the update very early use it directly through the Spotify client.
We will watch further ;)

**Reply by amd64fox** [reply]
Yes, this is where they get the early version, but the region is important, the region's account and perhaps something else to get it here.

The interception of the link has also failed so far for me

But I found a [website](https://snapcraft.io/spotify/embedded?channels=true) with a repository for Linux. A new version appeared on it quite early. I put him under surveillance to clarify this.

**Comment by revepastrop** [comment]
for the record, latest version 1.1.89.862 should be avoided if you're a playlist maker, in addition to the silly progress bar bug, there's the much more annoying "can't scroll while moving tracks in playlist" bug, which is a pain in the ass when new tracks get automatically added at the bottom.

**Reply by amd64fox** [reply]
Have you found out which version is the most stable for you?

**Reply by revepastrop** [reply]
1.1.87.612 seems free of those two bugs so I'll stick with that one for now :)

**Reply by amd64fox** [reply]
Yes, I confirm that version `1.1.87` is stable enough, let's hope that in the future version `1.1.90` these errors will be corrected.
